common/app/dev/DevParametersHttpRequestHandler.scala

```diff
@@ -17,7 +17,14 @@ class DevParametersHttpRequestHandler(
     configuration: HttpConfiguration,
     filters: HttpFilters,
     context: ApplicationContext,
-) extends DefaultHttpRequestHandler(webCommands, optionalDevContext, router, errorHandler, configuration, filters)
+) extends DefaultHttpRequestHandler(
+      webCommands,
+      optionalDevContext,
+      () => router,
+      errorHandler,
+      configuration,
+      filters,
+    )
```

Comment:
To address error:

```
/Users/Ioanna_Kokkini/code/frontend/common/app/dev/DevParametersHttpRequestHandler.scala:22:11: constructor DefaultHttpRequestHandler in class DefaultHttpRequestHandler is deprecated (since 2.9.0): Use the main DefaultHttpRequestHandler constructor
[error] ) extends DefaultHttpRequestHandler(webCommands, optionalDevContext, router, errorHandler, configuration, filters)
[error]           ^
[error] one error found
```